src/app/shared/components/checkbox-input/checkbox-input-field.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
```

Comment:
Please use styled checkbox from argo-ui library. Also CheckboxInput wrapper is not required. To avoid React warning about uncontrolled input we need to make sure `checked` prop is not null:

```
import { Checkbox } from 'argo-ui';
import * as React from 'react';
import * as ReactForm from 'react-form';

export const CheckboxInputField = ReactForm.FormField((props: { fieldApi: ReactForm.FieldApi, className: string, checked: boolean }) => {
    const { fieldApi: {getValue, setValue}} = props;

    return (
        <Checkbox checked={!!getValue()} onChange={setValue} />
    );
});
```